web/beacon-app/src/components/layout/ProfileAvatar/ProfileAvatar.tsx

```diff
@@ -0,0 +1,15 @@
+import { getInitials } from '@/utils/strings';
+
+type ProfileAvatar = {
+  name: string;
+};
+
+const ProfileAvatar = ({ name }: ProfileAvatar) => {
```

Comment:
The sidebar menu currently displays the gravatar from the profile picture. In order to not display the same image twice, this avatar was created that only displays the user's initials.